dev/ci/scripts/utils/run_all_experiments.sh

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/env bash
```

Comment:
Can we discuss this script and its use outside of the context of this PR?

Reply:
Sure, np.  I'll remove this one too.